jacd.c

```diff
@@ -140,6 +141,28 @@ static int lookup_name(uint64_t name)
 
 }
 
+static const char *canaddr2str(const struct sockaddr_can *paddr)
+{
```

Comment:
Is this the same or similar to https://github.com/linux-can/can-utils/blob/j1939/libj1939.c#L149 `libj1939_addr2str()`?

Reply:
Yes, exactly the same. We could move that into a separate file, maybe "canhelper" or something similar?

Reply:
What about `libj1939.c` :)

Reply:
Argh, sorry, I misread that. I actually took it from testj1939, will replace it in this PR as well as in testj1939. Yay, less duplicate code :)

Reply:
👍